test/io/test_io_image.py

```diff
@@ -28,6 +29,40 @@ def create_random_img8_torch(height: int, width: int, channels: int, device=None
     return (torch.rand(channels, height, width, device=device) * 255).to(torch.uint8)
 
 
+def _download_image(url: str, filename: str = "") -> str:
+    # TODO: move this to testing
+
+    filename = url.split("/")[-1] if len(filename) == 0 else filename
+    # Download
+    bytesio = io.BytesIO(requests.get(url, timeout=60).content)
+    # Save file
+    with open(filename, "wb") as outfile:
+        outfile.write(bytesio.getbuffer())
+
+    return filename
+
+
+@pytest.fixture(scope='class')
+def png_image(tmp_path_factory):
+    url = "https://github.com/kornia/data/raw/main/simba.png"
+    filename = tmp_path_factory.mktemp("data") / "image.png"
+    filename = _download_image(url, str(filename))
+    return filename
+
+
+@pytest.fixture(scope='class')
+def jpg_image(tmp_path_factory):
+    url = "https://github.com/kornia/data/raw/main/crowd.jpg"
+    filename = tmp_path_factory.mktemp("data") / "image.jpg"
+    filename = _download_image(url, str(filename))
+    return filename
```

Comment:
return as Path ?

Reply:
yep, this way we can load from disk within kornia api

Reply:
now it returns a `str` not `Path`

Reply:
true, got it